src/main/java/dev/blachut/svelte/lang/codeInsight/SvelteReferenceSearch.kt

```diff
@@ -10,6 +11,9 @@ import com.intellij.util.QueryExecutor
 
 class SvelteReferenceSearch : QueryExecutor<PsiReference, ReferencesSearch.SearchParameters> {
     override fun execute(queryParameters: ReferencesSearch.SearchParameters, consumer: Processor<in PsiReference>): Boolean {
+        if (!ApplicationManager.getApplication().isReadAccessAllowed) {
```

Comment:
it is better to wrap the code bellow into a read action ReadAction.run() or ReadAction.compute()

Reply:
Thank you, I will try that